internal/test/wmcb/wmcb_test.go

```diff
@@ -113,6 +147,25 @@ func (vm *wmcbVM) runTestBootstrapper(t *testing.T) {
 	require.NoError(t, err, "TestBootstrapper failed")
 }
 
+// runTestConfigureCNI performs the required setup and runs the configure-cni tests
+func (vm *wmcbVM) runTestConfigureCNI(t *testing.T) {
+	err := vm.initializeHybridOverlayBinary()
+	require.NoError(t, err, "error initializing files required for TestConfigureCNI")
+
+	node, err := framework.GetNode(vm.GetCredentials().GetIPAddress())
+	require.NoError(t, err, "unable to get node object for VM")
+
+	err = vm.handleHybridOverlay(node.GetName())
+	require.NoError(t, err, "unable to handle hybrid-overlay")
+
+	// It is guaranteed that the hybrid overlay annotations are present as we have already checked for it
+	err = vm.initializeTestConfigureCNIFiles(node.GetAnnotations()[test.HybridOverlaySubnet])
```

Comment:
I was stuck here trying to understand how this function call worked. Moving the `GetAnnotations()` outside of the function call and assigning it to a variable before indexing it would help readability.